src/fides/api/schemas/connection_configuration/connection_secrets_base_aws.py

```diff
@@ -14,7 +14,6 @@ class BaseAWSSchema(ConnectionConfigSecretsSchema):
     auth_method: AWSAuthMethod = Field(
         title="Authentication Method",
         description="Determines which type of authentication method to use for connecting to Amazon Web Services. Currently accepted values are: `secret_keys` or `automatic`.",
-        default=AWSAuthMethod.SECRET_KEYS,
```

Comment:
the default value here was making it so that it didn't show up in the `required` array, even though it's a required field. as far as i can tell, there's no two ways about this with pydantic - you can't really declare a field with a default value _and_ consider it required, or you have to consider _all_ default fields as required (which we don't want to do).

the default value here is only a nice to have - we're just guessing on what people prefer, anyway...we don't really know. on the other hand, i think showing this field as `required` is more important, since it's required.